lib/models/movie.dart

```diff
@@ -0,0 +1,34 @@
+class Movie {
+  final String title;
+  final String originalTitle;
+  final String overview;
+  final DateTime? releaseDate;
+  final String poster;
+  final String backDrop;
+  final double rate;
+  final List<String> genres;
+
+  Movie({
+    this.releaseDate,
+    required this.originalTitle,
+    required this.overview,
+    required this.title,
+    required this.backDrop,
+    required this.poster,
+    required this.rate,
+    required this.genres,
+  });
+
+
+  factory Movie.fromJson({required Map<String, dynamic> json, required List<String> genres, required String baseImageUrl}) {
+    return Movie(
+        originalTitle: json['original_title'],
+        overview: json['overview'],
+        title: json['title'],
+        backDrop: baseImageUrl + json['backdrop_path'],
+        poster: baseImageUrl + json['poster_path'],
```

Comment:
Instead to do this, made a method that return the complete url for backdrop_path and also poster_path, something like this:
```
String get fullBackdropPath => '$baseImageUrl$backdrop';
String get fullPosterPath => '$baseImageUrl$poster';
```

Reply:
Nice! Done